src/features/metadata-instance-editor/CascadePolicy.js

```diff
@@ -126,6 +141,17 @@ const CascadePolicy = ({
                                 <FormattedMessage {...messages.aiAutofillLearnMore} />
                             </Link>
                         </div>
+                        <div className="metadata-cascade-ai-agent-selector">
+                            <TooltipProvider>
```

Comment:
`TooltipProvider` shouldn't be needed here since the parent apps should have the providers at the root level. if this is just for the test, then `renderComponent` in the test file can wrap the correct providers

Reply:
we can address this in the follow up

Reply:
For some reason I was getting an error that it needed a TooltipProvider. I'll look into this on Monday. I know EUA already is wrapped in a TooltipProvider which is why I was confused